tests/inspectdb/tests.py

```diff
@@ -381,9 +382,9 @@ def test_foreign_data_wrapper(self):
                     sepal_length real,
                     sepal_width real
                 ) SERVER inspectdb_server OPTIONS (
-                    filename '/dev/null'
+                    filename '{}'
```

Comment:
```suggestion
                    filename %s
```

Reply:
Shouldn't we wrap the %s inside a string? Might be wrong here